### ISSUE TITLE
[Mage] Increased cost of Arcane Blast from Arcane Charge seems incorrect.

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#1454]
Originally reported on Google Code with ID 1455

```
While I can't be 100% sure (more like 98%), simulationcraft seems to be using an incorrect
mana cost for Arcane Blast based on the number of stacks of Arcane Charge present,
which drastically alters results.

With patch 5.1, the Arcane Charge (de)buff was changed to increase the cost of Arcane
Blast by 75% per stack, down from 125%. The in-game tooltip for Arcane Blast claims
the cost increase is still 125%, however the tooltip for the Arcane Charge (de)buff
correctly states that the cost is increased by 75% times the number of stacks currently
present. Additionally, the numerical mana cost listed in Arcane Blast's tooltip becomes
7875 when the player has 1 stack of Charge, which is exactly a 75% increase.

IN SHORT: I am 100% positive that the true in-game cost of ABlast is increased by 75%
per stack of Charge, and I am 98% positive that simulationcraft still uses the old
125%.
```

Reported by `mkeeton4` on 2012-11-30 22:40:59
